client/tests/sdk/test_shared.py

```diff
@@ -162,7 +162,7 @@ def error_unencrypted_reply(self):
         s = self.api.get_sources()[0]
         with pytest.raises(ReplyError) as err:
             self.api.reply_source(s, "hello")
-        assert str(err.value) == "'bad request'"
+        assert str(err.value) == "'Bad request'"
```

Comment:
Compare:

https://github.com/freedomofpress/securedrop-client/blob/9746dce111d4f3d70ae830c50c3e96d5be38a3ea/client/securedrop_client/sdk/__init__.py#L192
https://github.com/freedomofpress/securedrop-client/blob/9746dce111d4f3d70ae830c50c3e96d5be38a3ea/client/securedrop_client/sdk/__init__.py#L714

Reply:
Thanks, I've changed the latter to be uppercase and added a fixme for the first on why it's a ReplyError versus a more general HTTP error class.